packages/dd-trace/test/appsec/index.fastify.plugin.spec.js

```diff
@@ -536,6 +520,93 @@ withVersions('fastify', 'fastify', '>=2', version => {
       })
     })
   })
+
+  describe('Suspicious request blocking - multipart', () => {
+    withVersions('fastify', '@fastify/multipart', (multipartVersion, _, multipartLoadedVersion) => {
+      let server, uploadSpy, axios
+
+      before(function () {
+        // @fastify/multipart is not compatible with Fastify 2.x
+        if (semver.intersects(fastifyLoadedVersion, '2')) {
+          this.skip()
+        }
+
+        // 6.x doesn't work for this specific Fastify patch version.
+        if (semver.intersects(fastifyLoadedVersion, '3.9.2') && semver.intersects(multipartLoadedVersion, '>6')) {
+          this.skip()
+        }
+
+        if (semver.intersects(fastifyLoadedVersion, '>3') && semver.intersects(multipartLoadedVersion, '<7.0.0')) {
+          this.skip()
+        }
```

Comment:
are you sure this is what you mean ?
`>6` won't match 6.0.0, but it's roughly equivalent to >=7.0.0 no ?
same thing for `>3`